### ISSUE TITLE
Entity Detector Filter Configuration?

### ISSUE THREAD

**Issue by eslachance** [brandon3055/Draconic-Evolution#457]
I think maybe this is a very recent block addition, I can't actually find any information on it. It seems straightfoward enough, but I can't get it to work. Most notably, when I click on the Filter configuration I get a grey box with a Close button and nothing else. So I can't configure the detector to really *do* anything.

**Comment by brandon3055** [comment]
Can you give me a screenshot please. Sounds like it may be broken. Also which one are you using? there are 2 tiers

**Comment by SOL3675** [comment]
I have a same situation. Here is the entity detector's configuration GUI.
![2016-11-29_11 41 14](https://cloud.githubusercontent.com/assets/16431933/20694762/d3948f80-b62a-11e6-9c94-91845dd66965.png)

The advanced entity detector looks like fine, but I clicked the +enitity button, I've got crushed...
[http://pastebin.com/8WWWhsH7](url)
I'm using All the Mods pack with Draconic-Evolution-1.10.2-2.0.3.137.
